project/hidp_django_admin_sandbox/settings.py

```diff
@@ -0,0 +1,283 @@
+
+TIME_ZONE = "Europe/Amsterdam"
+
+USE_I18N = True
+
+USE_TZ = True
+
+LOCALE_PATHS = [PROJECT_DIR / "locale"]
+
+# Media files (user uploads)
+
+MEDIA_URL = "/media/"
+
+if DEBUG:
+    # MEDIA_ROOT defaults to VAR_DIR/public/media/ in debug mode
+    MEDIA_ROOT = Path(
+        config.getliteral("app", "media_root", fallback=VAR_DIR / "public/media")
+    )
+else:
+    # Require MEDIA_ROOT to be configured
+    MEDIA_ROOT = Path(config.getliteral("app", "media_root"))
+
+# File upload permissions
+# https://docs.djangoproject.com/en/4.2/ref/settings/#file-upload-permissions
+FILE_UPLOAD_PERMISSIONS = 0o644
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/4.2/howto/static-files/
+
+STATIC_URL = "/static/"
+
+STATICFILES_DIRS = [
+    FRONTEND_DIR / "static",
+]
+
+if DEBUG:
+    # STATIC_ROOT defaults to VAR_DIR/public/static/ in debug mode
+    STATIC_ROOT = Path(
+        config.getliteral("app", "static_root", fallback=VAR_DIR / "public/static")
+    )
+else:
+    # Require STATIC_ROOT to be configured
+    STATIC_ROOT = Path(config.getliteral("app", "static_root"))
+
+# File storage
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "staticfiles": {
+        "BACKEND": "hidp_django_admin_sandbox.storage.staticfiles.SelectiveManifestStaticFilesStorage",  # noqa: E501, RUF100
+    },
+}
+
+# Email backends
+EMAIL_BANDIT = config.getliteral("app", "email_bandit", fallback=False)
+
+if DEBUG:
+    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+else:
+    EMAIL_HOST = config.getliteral("app", "email_host", fallback="localhost")
+    EMAIL_PORT = config.getliteral("app", "email_port", fallback=25)
+    EMAIL_HOST_USER = config.getliteral("app", "email_host_user", fallback="")
+    EMAIL_HOST_PASSWORD = config.getliteral("app", "email_host_password", fallback="")
+    EMAIL_USE_TLS = config.getliteral("app", "email_use_tls", fallback=False)
+    EMAIL_USE_SSL = config.getliteral("app", "email_use_ssl", fallback=False)
+
+    if EMAIL_BANDIT:
+        EMAIL_BACKEND = "bandit.backends.smtp.HijackSMTPBackend"
+        BANDIT_EMAIL = config.getliteral("app", "bandit_email")
+        BANDIT_WHITELIST = config.getliteral(
+            "app", "bandit_whitelist", fallback=["leukeleu.nl"]
+        )
+
+DEFAULT_FROM_EMAIL = config.getliteral("app", "default_from_email")
+
+# Django GDPR
+DJANGO_GDPR_YML_DIR = PROJECT_DIR.parent
+
+
+class InternalIPList:
+    """
+    A fake list that checks if a given ip address
+    is local (loopback) or internal (private)
+    """
+
+    def __contains__(self, item):
+        address = ipaddress.ip_address(item)
+        return address.is_loopback or address.is_private
+
+
+if DEBUG:
+    INTERNAL_IPS = InternalIPList()
+
+# Hello, ID Please
+
+LOGIN_URL = "hidp_accounts:login"
+LOGIN_REDIRECT_URL = "/"
+LOGOUT_REDIRECT_URL = "/"
+
+OTP_TOTP_ISSUER = "HIdP Django Admin Sandbox"
+
+REGISTRATION_ENABLED = False
```

Comment:
`REGISTRATION_ENABLED` can probably be better set to `True` so that we can also test that part?